android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -1179,19 +1180,31 @@ class JsonConverter {
          * Converts the [json] to a `RNUiConfig` object.
          */
         fun toPlayerViewConfig(json: ReadableMap) = PlayerViewConfig(
-            uiConfig = UiConfig.WebUi(
-                playbackSpeedSelectionEnabled = json.getMap("uiConfig")
-                    ?.getBooleanOrNull("playbackSpeedSelectionEnabled")
-                    ?: true,
-            ),
+            uiConfig = if (json.getMap("styleConfig")?.getBooleanOrNull("isUiEnabled") == false) {
+                UiConfig.Disabled
+            } else {
+                UiConfig.WebUi(
+                    playbackSpeedSelectionEnabled = json.getMap("uiConfig")
+                        ?.getBooleanOrNull("playbackSpeedSelectionEnabled")
+                        ?: true,
+                )
+            },
         )
 
+        private fun toUserInterfaceType(json: ReadableMap): UserInterfaceType =
+            when (json.getMap("styleConfig")?.getString("userInterfaceType")) {
+                "Subtitle" -> UserInterfaceType.Subtitle
+                "Bitmovin" -> UserInterfaceType.Bitmovin
+                else -> UserInterfaceType.Bitmovin
+            }
+
         /**
          * Converts the [json] to a `RNPlayerViewConfig` object.
          */
         fun toRNPlayerViewConfigWrapper(json: ReadableMap) = RNPlayerViewConfigWrapper(
             playerViewConfig = toPlayerViewConfig(json),
             pictureInPictureConfig = toPictureInPictureConfig(json.getMap("pictureInPictureConfig")),
+            userInterfaceType = toUserInterfaceType(json),
```

Comment:
I think you are mixing here unrelated configurations 🤔 
The `PlayerViewConfig` is an own type in the `ts` world.

I think this would require a `RNStyleConfigWrapper` (?)

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/316/commits/501f7bc8f2097f0f8970a0392e6413ae2c1c0875